src/index.scss

```diff
@@ -0,0 +1,18 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
This practice is quite common, but I would recommend slightly different style:

```css
*,
*::before,
*::after {
  box-sizing: inherit;
}

html {
  box-sizing: border-box;
}
```

This way you still can easily override box-sizing for a component tree (you ever need that). It's not really crucial, but it provides flexibility for free so why not use it? 😄